source/AuthManager.ts

```diff
@@ -112,16 +129,16 @@ export class AuthManager {
         }.bind(this));
     }
 
-    public configureIamCredentials() {
+    public getIamCredentials(): Promise<AWS.CognitoIdentityCredentials> {
```

Comment:
Returning the credential in a promise allows the consumer to use it however they wish